app/cardrecord.js

```diff
@@ -28,23 +28,27 @@ class CardRecorder extends MyTrello {
                 self.deleteCurrentComment(comments).then(function(resp) {
                     var updateActions = _.filter(card.actions, {type: "updateCard"});
                     var createAction = _.filter(card.actions, {type: "createCard"});
+                    var now = moment();
                     var hasMoved = false;
+                    var daysSinceUpdate = false;
                     if(typeof updateActions !== "undefined"){
                       hasMoved = self.hasMovedCheck(updateActions);
+                      daysSinceUpdate = now.diff(moment(updateActions[0].date), 'days');
                     }
+                    var totDays = (comments !== "undefined")? self.calcTotalDays(comments, now) : 0;
+                    console.log("total days:"+totDays);
                     var lastMove = self.findLastMoveDateFromComments({commentList: comments, "actionList": updateActions, "createActionDate": createAction.date});
```

Comment:
So that's how it was, but because human's don't always move it on the day that they even happened, this gives a more fully-proof way for the PMs/others to edit the comment, and have that change be reflected in the manifest. See #47 for more about why this was added. I agree that is more technically efficient, but this is more user-friendly.